### ISSUE TITLE
Automatically close clipboard when pasting and change back button position

### ISSUE THREAD

**Issue by NikunjKhangwal** [futo-org/android-keyboard#470]
### Use Cases

It's a bit inconvenient to close clipboard manually everytime you pasted something and also the back button in clipboard is on left side.

### Proposal

An option to automatically close clipboard and switch back to keyboard when a copied entry is pasted so we don't have to go back manually. The reason why this should be optional is because in a lot of cases you want to paste multiple items in same field as well so it'd be annoying if it's always enabled.

The next similar suggestion is probably just me but I want to highlight it anyway. I understand if it's a unreasonable request and don't get implemented. Because I keep the clipboard pinned it would be nice to have the back button on the right side for same muscle memory and easier one hand usage.

![IMG_20240815_003132](https://github.com/user-attachments/assets/7c94ae71-7611-4e59-99d9-e791485f9186)

**Comment by The-Quantum-Alpha** [comment]
While I understand the inconvenience you've described, I have to point out that the current clipboard behavior allows pasting multiple items in the same field. An automatic closure might disrupt this workflow. As for the back button position, it's a common design pattern for Android keyboards and changing it might lead to inconsistencies. I am content with its current behavior, but it may be an option within the keyboard settings.

**Comment by NikunjKhangwal** [comment]
> I have to point out that the current clipboard behavior allows pasting multiple items in the same field. An automatic closure might disrupt this workflow. 

Yeah that's why i thought it should be optional. Maybe try implementing if it works out. If it doesn't that's fine too.

> As for the back button position, it's a common design pattern for Android keyboards and changing it might lead to inconsistencies.

Totally Fair

**Comment by arncor** [comment]
Hi, I used swiftkey for a long time before switching to FUTO and they have that behavior : auto-close the clipboard after pasting (and I miss it :)) 

Also, the first time you opened the keyboard after copying some text, the clipboard automatically opened (and I think it's great).
